platform/viewer/public/config/default.js

```diff
@@ -1,72 +1,116 @@
-window.config = {
-  // default: '/'
-  routerBasename: '/',
-  extensions: [],
-  showStudyList: true,
-  filterQueryParam: false,
-  servers: {
-    dicomWeb: [
+window.config = ({ servicesManager, dependencies }) => {
+  const { merge } = dependencies;
+
+  const allToolsProps = Object.assign(
+    ...[
+      'Bidirectional',
+      'Length',
+      'Angle',
+      'FreehandRoi',
+      'EllipticalRoi',
+      'CircleRoi',
+      'RectangleRoi',
+      'ArrowAnnotate',
+    ].map(tool => ({
+      [tool]: {
+        configuration: {
+          /* Add common tool configurations here. */
+        },
+      },
+    }))
+  );
+
+  /* Add specific tools configuration here. */
+  const specificToolsProps = {
+    /* ArrowAnnotate: {
+      configuration: {
+        ...
+      },
+    }, */
+  };
```

Comment:
While `allToolProps` and `specificToolProps` are nice guides for how to do this, now that we've offloaded to the extension, can these be removed?

Reply:
You right! removed. Thanks. 👍